scripts/devbox-init-hook.sh

```diff
@@ -2,29 +2,16 @@
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 SRC_DIR="$(dirname $DIR)"
 
-# Set install path
-INSTALL_DIR="$HOME/.local/bin"
-mkdir -p "$INSTALL_DIR"
-
-# Check if envsubst.old exists
-if [[ ! -f "/usr/bin/envsubst.old" ]]; then
-    echo "envsubst.old does not exist. Proceeding with installation..."
+# Set install path to devbox packages directory
+INSTALL_DIR="$DEVBOX_PACKAGES_DIR/bin"
 
+if [[ ! -f "$INSTALL_DIR/envsubst" ]]; then
     ENVSUBST_VERSION="v1.4.3"
     URL="https://github.com/a8m/envsubst/releases/download/$ENVSUBST_VERSION/envsubst-$(uname -s)-$(uname -m)"
     echo "Downloading: $URL"
-
-    curl -sL $URL -o $INSTALL_DIR/envsubst
-    chmod +x "$INSTALL_DIR/envsubst"
-
-    # Replace existing envsubst if present
-    if [[ -f "/usr/bin/envsubst" ]]; then
-        sudo mv /usr/bin/envsubst /usr/bin/envsubst.old
-        sudo ln -s "$INSTALL_DIR/envsubst" /usr/bin/envsubst
-        echo "Replaced existing envsubst with the new one at /usr/bin/envsubst"
-    else
-        touch /usr/bin/envsubst.old
-    fi
+    echo "Installing github.com/a8m/envsubst into Devbox shell"
```

Comment:
We have a target for envsubst in the tools makefile that we could use instead: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/635e9934270e58096e3ea59d9b8b88c0e0a546f7/hack/tools/Makefile#L89